From 33b4635eefae570c40fa4c5f2016321c371ae22a Mon Sep 17 00:00:00 2001
From: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
Date: Sun, 10 Apr 2011 22:08:14 -0300
Subject: [PATCH 2/3] panda: clocks: Disable slimbus and pad_clks

The slimbus and pad_clks if left enabled in x-loader
would gate abe power domain from transitioning to RET.

These should be enabled by the audio driver (if present)
in the kernel.

Patch based on the SDP one from dev.omapzoom.org (8876b861) by
Rajendra Nayak <rnayak@ti.com>

Signed-off-by: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
---
 board/omap4430panda/clock.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/board/omap4430panda/clock.c b/board/omap4430panda/clock.c
index 27f7d92..d9a925f 100644
--- a/board/omap4430panda/clock.c
+++ b/board/omap4430panda/clock.c
@@ -322,9 +322,6 @@ static void configure_abe_dpll(u32 clk_index)
 	/* Select sys_clk as ref clk for ABE dpll */
 	sr32(CM_ABE_PLL_REF_CLKSEL, 0, 32, 0x0);
 
-	/* Enable slimbus and pad clocks */
-	sr32(CM_CLKSEL_ABE, 0, 32, 0x500);
-
 	/* Unlock the ABE dpll */
 	sr32(CM_CLKMODE_DPLL_ABE, 0, 3, PLL_MN_POWER_BYPASS);
 	wait_on_value(BIT0, 0, CM_IDLEST_DPLL_ABE, LDELAY);
-- 
1.7.4.1

